Parent/Parent/Dashboard/DashboardViewController.swift

```diff
@@ -138,7 +141,7 @@ class DashboardViewController: UIViewController, CustomNavbarProtocol {
         bbi.tintColor = .white
         navigationItem.leftBarButtonItem = bbi
         bbi.accessibilityIdentifier = "Dashboard.profileButton"
-
+        bbi.accessibilityLabel = NSLocalizedString("Settings", comment: "")
```

Comment:
Can we add the badge number for inbox to this label. Perhaps something like `2 unread conversations`.